sources/packages/api/src-sql/Institution/Create-Institution.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS institutions(
+  id SERIAL PRIMARY KEY, 
+  legal_operating_name VARCHAR(64) NOT NULL, 
+  operating_name VARCHAR(64) NULL,  
+  primary_phone VARCHAR(64) NOT NULL,  
+  primary_email VARCHAR(64) NOT NULL,  
+  institution_type VARCHAR(64) NOT NULL,  
```

Comment:
I am not considering this field in the UI.
@abschwenker, are we good to add this field or are there some discussions about it?

Reply:
Confirmed with ticket - Please leave out for now

Reply:
Done